### PR TITLE
Adding installation of vscode in install phase

### PR DESCRIPTION
**Pull request by gaetbout**
Make install will now also build and compile the vscode extension
Also updated argent_account to pass tests due to SW changes

### CODE REVIEW

**Review by sgc-code — APPROVED** [review]
nice!

File: Makefile [hunk 1]
```diff
@@ -7,13 +7,18 @@ INSTALLATION_FOLDER=./cairo
 SOURCE_FOLDER=./contracts
 
 install: 
+	$(MAKE) simple-install
+	$(MAKE) vscode
+
+make simple-install:
```
Comment (delaaxe) [inline Makefile:13]:
```suggestion
make install-cairo:
```
?

Comment (gaetbout) [reply Makefile:13]: Much better naming indeed!

**Review by Leonard-Pat — APPROVED** [review]

**Comment by delaaxe** [comment]
needs rebase

**Review by delaaxe — APPROVED** [review]

**Pull request merged**